sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/StorageImplUtils.java

```diff
@@ -296,29 +293,6 @@ public static String emptyIfNull(String value) {
         return value == null ? "" : value;
     }
 
-    /**
-     * Reads data from an input stream and writes it to an output stream.
-     * @param source {@link InputStream source}
-     * @param writeLength The length of data to write.
-     * @param destination {@link OutputStream destination}
-     * @throws IOException If an I/O error occurs.
-     */
-    public static void copyToOutputStream(InputStream source, long writeLength, OutputStream destination)
```

Comment:
@alzimmermsft Do we want to keep this around for the same reason we did with the sas credential policy pr--in case someone upgrades common but not blob?